server/api.js

```diff
@@ -91,4 +91,40 @@ router.get("/laptop_donation", async (req, res) => {
 	}
 });
 
+// post laptop assignment
+router.post("/laptop_assignment", (req, res) => {
+	let laptopRequestsId = req.body.laptop_request_id;
+	let laptopDonationId = req.body.laptop_donation_id;
+	let status = req.body.status;
+	let assignmentId = req.params.laptop_assignment;
+	const query =
+		" insert into laptop_assignment (laptop_request_id, laptop_donation_id, status) values ($1, $2, $3 )";
+
+	db.query(query, [laptopRequestsId, laptopDonationId, status])
+		.then(() => res.send(`assignmentId ${assignmentId} posted!`))
```

Comment:
you need to get the id back from the rows of the query. Something like `.then((queryResult) => res.send(res.rows[0]))`

Reply:
done